mmv1/third_party/terraform/services/datafusion/resource_data_fusion_instance_test.go

```diff
@@ -207,3 +213,40 @@ resource "google_data_fusion_instance" "basic_instance" {
 }
 `, context)
 }
+
```

Comment:
This test is basically not doing anything.
We should be testing creation with deletionProtection true and then the delete fails with an error.

Reply:
done